pkg/artifactory/resource/user/resource_artifactory_user_test.go

```diff
@@ -5,182 +5,234 @@ import (
 	"net/http"
 	"testing"
 
-	"github.com/hashicorp/terraform-plugin-sdk/v2/helper/resource"
-	"github.com/hashicorp/terraform-plugin-sdk/v2/terraform"
+	"github.com/hashicorp/terraform-plugin-framework/providerserver"
+	"github.com/hashicorp/terraform-plugin-go/tfprotov5"
+	"github.com/hashicorp/terraform-plugin-testing/helper/resource"
+	"github.com/hashicorp/terraform-plugin-testing/terraform"
 	"github.com/jfrog/terraform-provider-artifactory/v7/pkg/acctest"
-	"github.com/jfrog/terraform-provider-shared/test"
-	"github.com/jfrog/terraform-provider-shared/util"
+	"github.com/jfrog/terraform-provider-artifactory/v7/pkg/artifactory/provider"
+	"github.com/jfrog/terraform-provider-shared/testutil"
+	utilsdk "github.com/jfrog/terraform-provider-shared/util/sdk"
 	"github.com/jfrog/terraform-provider-shared/validator"
+	//"github.com/mitchellh/go-testing-interface"
 )
 
-func TestAccUserPasswordNotChangeWhenOtherAttributesChangeGH340(t *testing.T) {
-	id := test.RandomInt()
-	name := fmt.Sprintf("user-%d", id)
-	fqrn := fmt.Sprintf("artifactory_user.%s", name)
+var (
+	// these will be set by the goreleaser configuration
+	// to appropriate values for the compiled binary
+	version = "test"
```

Comment:
This won't be necessary once we change how to set the provider's version string for framework.

Reply:
Removed from all tests.

Reply:
I'm setting the test version in the go run parameters now.